src/qt/walletmodel.cpp

```diff
@@ -137,12 +136,6 @@ void WalletModel::updateBalance()
     }
 }
 
-void WalletModel::updateTransaction()
```

Comment:
In commit "[Qt] remove unused polling code"

I think it would be good to squash this commit into "[Qt] use the BalancesChanged signal instead of a 250ms poll timer." It's confusing to see code that checks the `fForceCheckBalanceChanged` variable and some code that sets it removed, while other code that sets it sticks around.